src/App.jsx

```diff
@@ -14,28 +14,46 @@ import ProductDetails from '@pages/ProductDetails';
 import ProductList from '@pages/ProductList';
 import ProductSearch from '@pages/ProductSearch';
 
+/**
+ * `Theme
+ */
+
+const lightTheme = createTheme({
```

Comment:
There is an issue that makes some MUI elements to disappear caused the the primary and secondary properties.
If you remove them it works as expected